motors:
  - name: PAN
    v_max: 10.0
    a_max: 6.45
  - name: LIFT
    v_max: 10.0
    a_max: 6.0